### ISSUE TITLE
Publish library to pip

### ISSUE THREAD

**Issue by buremba** [metriql/metriql-lookml#5]
We should publish the library to PyPI and let people install the library and run them locally as follows:

```
pip install metriql-looker
metriql-looker [ARG]
```

**Comment by buremba** [comment]
waiting for #10

**Comment by UmarMajeed-Rana** [comment]
@buremba as #10 is merged as released can we close this issue?

**Comment by buremba** [comment]
Yes, we already published the package: https://pypi.org/project/metriql-lookml/ We were focusing on making the Google Data Studio integration stable for the last couple of weeks, I will be focusing on Looker integration later when we have a prospect who is willing to try out the integration in production. The challenge that I faced with the Looker integration is that you need to go through their enterprise sales cycle to test metriql and while I managed to get one account, they removed the access after 5 days. :(
